.gitignore

```diff
@@ -230,4 +230,9 @@ pip-log.txt
 #Mr Developer
 .mr.developer.cfg
 MediaBrowser.WebDashboard/dashboard-ui/.idea/
-.vscode
\ No newline at end of file
+.vscode
+
+#############
+## Node (NPM/Yarn)
+#############
```

Comment:
consistent comments would be nice :p the one above this has no border and no space after the hash character

Reply:
Yeah it's a mess :p. Tons of unrelated ignores too so I'll probably just clean all of it up